### ISSUE TITLE
Correct way to access 'views'

### ISSUE THREAD

**Issue by knightlightning** [snipe/snipe-it#5619]
**Server (please complete the following information):**
 - Snipe-IT Version v4.4.1 - build 3636 (master)
 - OS: [e.g. Ubuntu, CentOS] Ubuntu 18.04
 - Web Server: [e.g. Apache, IIS] Apache 2.4
 - PHP Version 7.2.5-0ubuntu0.18.04.1

**Is your feature request related to a problem? Please describe.**
Can't find any links in the web UI that allow me (as superadmin) to access some pages. For instance, if I click on 'ready to deploy' label of the corresponding asset it'll redirect me to '/statuslabels/2' page. Now, in order to actually edit status labels, I have to go to the '/statuslabels' page which I cannot find how to get to rather than to put that URL directly in the browser's address bar.

**Describe the solution you'd like**
Provide some means to access such views.

**Describe alternatives you've considered**

**Additional context**

**Comment by snipe** [comment]
Nah, not your fault. We have two gear icons in the UI, and that's a bit confusing. :-/
